gazebo_ros/test/ros_network/ros_network_default.test

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0"?>
+<launch>
+  <include file="$(find gazebo_ros)/launch/empty_world.launch">
+    <arg name="paused"   value="false"/>
+    <arg name="gui"      value="false"/>
+    <arg name="headless" value="false"/>
+    <arg name="debug"    value="false"/>
+    <arg name="enable_ros_network" value="true" />
```

Comment:
Perhaps this argument should be excluded entirely in this test to ensure the default behavior is enabled.